core/connection/session_storage.go

```diff
@@ -41,9 +43,9 @@ func (repo *SessionStorage) Save(se Session) error {
 }
 
 // Update updates specified fields of existing session by id
-func (repo *SessionStorage) Update(sessionID session.ID, duration int, dataStats stats.SessionStats) error {
+func (repo *SessionStorage) Update(sessionID session.ID, TimeUpdated time.Time, dataStats stats.SessionStats) error {
 	// update two fields by sessionID
-	se := Session{SessionID: sessionID, Duration: duration, DataStats: dataStats}
+	se := Session{SessionID: sessionID, TimeUpdated: TimeUpdated, DataStats: dataStats}
```

Comment:
Do we save session status - NEW, IN Progress, COMPLETED ? How can I tell which session is finished which is ongoing?

Reply:
Is there a need for session status in history list? This can be added as feature later as we get feedback from using this endpoint

Reply:
imho NEW and COMPLETED should be enough. New session will have TimeStarted set, completed session will have TimeUpdated set?

Reply:
added statuses New and Completed